plugins/source/aws/client/tableoptions/costexplorer.go

```diff
@@ -0,0 +1,45 @@
+package tableoptions
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/costexplorer"
+	"github.com/cloudquery/plugin-sdk/v3/caser"
+)
+
+type CostExplorerAPIs struct {
+	GetCostAndUsageOpts []CustomGetCostAndUsageInput `json:"get_cost_and_usage,omitempty"`
+}
+
+type CustomGetCostAndUsageInput struct {
+	costexplorer.GetCostAndUsageInput
```

Comment:
Maybe also add a `name` here people can fill in and use as a 'preset' while accessing data (if they have multiple inputs?)
It would be better than keeping track of what `input_hash` corresponds to what input, or selecting data directly from `input_json` (if it's more than start/end dates... which again since `GetCostAndUsageInput` doesn't contain a name, they'd have to match multiple fields)

Reply:
I see the issue you are raising, but I am concerned about deviating from the exact AWS defined input as it would make documentation and usage more complex. 

I think no matter what they would have to match on multiple fields because I don't think they can guarantee the `name` field would be unique enough as it would be 100% user defiend

Reply:
Why would they access by row number? I would expect users to parse the `input_json` column to ensure they are using the right input... I would make that column a PK, but not all destinations support JSON PKs...

Also we would have to name the input field something like `_cq_input_name` to ensure that we never have a collision with a field in an input

Reply:
ok let's revisit once/if it becomes a user-reported problem.